src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,137 @@
 package core.basesyntax;
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int INITIAL_CAPACITY = 16;
+    private static final int forNullKeyIndex = 0;
+    private static final double LOAD_FACTOR = 0.75;
+    private int threshold;
+    private int capacity;
```

Comment:
please try to rewrite your solution without those two fields, they are useless - capacity is just .length of the table, threshold is just product of load factor and capacity